oar/core/shipment.py

```diff
@@ -234,3 +289,38 @@ def get_nvrs(self) -> List[str]:
                 continue
                 
         return sorted(nvrs)
+
+    def add_qe_release_lead_comment(self, email: str) -> None:
+        """Add comment to all shipment merge requests identifying QE release lead
+        
+        Args:
+            email: Email address of QE release lead to look up in GitLab
+            
+        Raises:
+            ShipmentDataException: If email is invalid or username not found
+        """
+        if not email or not isinstance(email, str):
+            raise ShipmentDataException("Email must be a non-empty string")
+            
+        try:
+            gitlab_url = self._cs.get_gitlab_url()
+            token = self._cs.get_gitlab_token()
+            gl_server = GitLabServer(gitlab_url, token)
+            
+            username = gl_server.get_username_by_email(email)
+            if not username:
+                raise ShipmentDataException(f"No GitLab user found for email: {email}")
+                
+            comment = f"QE Release Lead from ERT is @{username}"
```

Comment:
In the future if the release could be maintained by sustaining team. Maybe we should use more general phrase.

Reply:
yes, remove ERT from the comment